task-library/AHV.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+##############################################
+# Name        : AHV.py
+# Author      : Calm Devops
+# Version     : 1.0
+# Description : Script will list all VMs available in a PC and update a VM
+##############################################
+import requests
+import json
+import re
+import time
+import sys
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class RestUtilException(Exception):
+    pass
+
+def _wrap(resp):
+    if resp.status_code == 200 :
+        try:
+           return resp.json()
+        except:
+           return None
+    else:
+        try:
+            resp_json = resp.json()
+            if 'message' in resp_json:
+                raise RestUtilException("Rest API request failed with error: %s" % resp_json['message'])
+        except Exception:
+            print(resp)
+            raise RestUtilException("Rest API request failed : %s" % resp.reason)
+
+def _do_get(url, cookies=None, params=None, auth=None, timeout=120):
```

Comment:
Please add doc_string which will detailed about function purpose, in args, return value etc..